packages/bind/src/DescendantBindingHandler.ts

```diff
@@ -2,6 +2,7 @@
 
 import { applyBindingsToDescendants } from './applyBindings'
 import { AsyncBindingHandler } from './BindingHandler'
+import { BindingResult } from "./BindingResult";
```

Comment:
should be `import type { BindingResult } ...`?

Reply:
Nice, it could also be a solution for the remaining global KO-defs, so we can move them into the associated modules in a second step. @mcselle FYI "Type-Only Imports": https://www.typescriptlang.org/docs/handbook/modules/reference.html

Reply:
Done.